Aztec/Classes/Public/Formatters/AttributeFormatter.swift

```diff
@@ -56,6 +56,12 @@ extension AttributeFormatter {
     func applicationRange(forRange range: NSRange, inString string: NSAttributedString) -> NSRange {
         return range
     }
+
+    /// The string to be used when adding attributes to an empty line.
+    ///
+    var placeholderForAttributedEmptyLine: String {
```

Comment:
Because `Extensions may not contain stored properties`, and `Static stored properties not yet supported in generic types` 😉